tests/serverpod_test_server/test_e2e_migrations/defaults/migrations_datetime_default_add_test.dart

```diff
@@ -150,7 +150,6 @@ void main() {
         var createMigrationExitCode =
             await MigrationTestUtils.createMigrationFromProtocols(
           protocols: targetStateProtocols,
-          force: true,
           tag: 'modify-existing-column',
         );
```

Comment:
Suggestion: This seem wrong, as discussed last call when going from a nullable to a none nullable column even with a default value we may fail the migration. This is because PostgreSQL is not writing the default values into existing null columns. 

Hence this operation can fail unless the programmer first migrates all null values.